Tests/AppTests/KarmaParserTests.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  KarmaParserTests.swift
+//  AppTests
+//
+//  Created by Jacob Wilson on 3/4/19.
+//
+
+@testable import App
+import XCTest
+
+final class KarmaParserTests: XCTestCase {
```

Comment:
I'm not seeing any tests for the format "@Jacob++ @allen++"

Reply:
Good catch....